bgc_data/bgc_phytoplankton_abundance_genus_data.sql

```diff
@@ -11,7 +11,8 @@ WITH grouped AS (
            substring(taxon_name, '^\w+') != substring(parent_name, '^\w+') AS genus_changed,
            sum(r.cell_l) AS cell_l
     FROM bgc_phyto_raw r LEFT JOIN bgc_phyto_changelog c USING (taxon_name)
-    WHERE r.genus IS NOT NULL
+    WHERE r.genus IS NOT NULL AND
+    r.genus NOT LIKE '%cf.%'
```

Comment:
Would be nice to indent this as you have done elsewhere...

Reply:
adjusted in the last commit